### PR TITLE
Portenta Machine Control: Hardware changes (Product Page + Datasheet) [PC-948, TC-374]

### PR DESCRIPTION
**Pull request by marqdevx**
## What This PR Changes
- Reflect changes from the new hardware version.

### CODE REVIEW

**Comment by jcarolinares** [comment]
[featured.png](https://github.com/arduino/docs-content/pull/559/files#diff-235d5f180a273f97ec7f542546b8602af3f25f77a363039e9f1c5460abcb6232) please change this image to the full product including the DIN bar supports

**Review by jcarolinares — CHANGES_REQUESTED** [review]
@marqdevx please take a look and accept the suggestions of the review if they are fine.

This is a WIP, due the web editor limitations we cannot comment the grey lines of the files, therefore one the suggestions are added, I will add commits to change the rest of the info

**Comment by marqdevx** [comment]
@jcarolinares Updated the `featured` picture
Should we update the features list, to a table?

**Review by jcarolinares — COMMENTED** [review]
Changes done. WIP thought new commits by @jcarolinares

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md [hunk 1]
```diff
@@ -145,228 +143,206 @@ Industry 4.0,  system integrators
...
+### Front-Ends
+There are two front ends on the board:
+- MAX31855KASA+T dedicated to thermocouples.
+- MAX31865ATP+T dedicated to PT100.
 
-- MAX31855KASA+T dedicated to thermocouples
-- MAX31865ATP+T dedicated to PT100
+The front ends are multiplexed to the three channels via:
+  - A single low-ohmic single-pole double-throw analog switch **NX3L4053HR,115** which is switching between one front end or the other.
+  - Three quadruple single pole single throw analog switches **TMUX1511RSVR** which are switching the active channel between the three available.
```
Comment (marqdevx) [inline content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md:225]: Align with the previous sentence
```suggestion
  - Three quadruple single-pole single-throw analog switches **TMUX1511RSVR** which are switching the active channel between the three available. 
```

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md [hunk 2]
```diff
@@ -145,228 +143,206 @@ Industry 4.0,  system integrators
...
+  - It can be used both as a host and as a device.
+  - ESD protection.
+
+### Half Speed Micro-USB Portenta Half-Speed USB Is Connected To The Micro-USB Connector
+  - Useful to program the Portenta board via a micro-usb cable.
+  - It can be used to power the Portenta board while the 24V power supply is off.
+  - ESD protection.
+
+### RTC
+The on-board real-time clock/calendar is the **PCF8563T/F4,118** which clock is provided by a dedicated external crystal oscillator:
```
Comment (marqdevx) [inline content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md:321]:
```suggestion
The on-board real-time clock/calendar is the **PCF8563T/F4,118** its clock is provided by a dedicated external crystal oscillator:
```

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md [hunk 3]
```diff
@@ -145,228 +143,206 @@ Industry 4.0,  system integrators
...
+  - A 100mF supercapacitor (**FC0V104ZFTBR24**) provides power to the **PCF8563T/F4,118** when the board power supply is disconnected. The RTC chipset will be powered by the supercapacitor for at least 48 hours.
```
Comment (marqdevx) [inline content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md:322]:
```suggestion
  - A 100mF super-capacitor (**FC0V104ZFTBR24**) provides power to the **PCF8563T/F4,118** when the board power supply is disconnected. The super-capacitor will supply the RTC chipset for at least 48 hours.
```

File: content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md [hunk 4]
```diff
@@ -145,228 +143,206 @@ Industry 4.0,  system integrators
...
-### Analog Output 
-Four independent analog output channels are available. Each of them a double low pass filter and a high current op amp arranged in a non-inverting topology with gain 3.3.
+### Analog Output
+Four independent analog output channels are available. Each of them has a double low-pass filter and a high-current op-amp arranged in a non-inverting topology with a gain of 3.3.
 
-![AnalogOutput](assets/MachineControlAnalogOutputScheme.png)
+![AnalogOutput](assets/MachineControlAnalogOutputScheme.svg)
 
-At each input of each channel a PWM from Portenta is provided filtered by a double low pass filter, obtaining a DC output with a small AC ripple. The signal is then fed to the channel non inverting amplifier which amplifies it by 3.3. 
+At each input of each channel, a PWM from the Portenta board is provided and filtered by a double low-pass filter, obtaining a DC output with a small AC ripple. The signal is then fed to the channel non-inverting amplifier which amplifies it by 3.3.
```
Comment (marqdevx) [inline content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/datasheet/datasheet.md:204]:
```suggestion
At each input of each channel, a PWM from the board is provided and filtered by a double low-pass filter, obtaining a DC output with a small AC ripple. The signal is then fed to the channel non-inverting amplifier which amplifies it by 3.3.
```

**Comment by marqdevx** [comment]
@jcarolinares 
I've just pushed some MD fixes, did some suggestions and I have some thoughts/issues:
* Isn't this duplicated? (4.5 Digital Programmable)
  image
* Why do we use A/D instead of ADC
* Analog Output (4.7) Should we use x3.3 instead 3.3 for the scale value? 
* Note information duplicated 
  image
* Please review the heading sentence `4.19 Half Speed Micro-USB Portenta Half-Speed USB Is Connected To The
Micro-USB Connector` 
* Analog Input section(4.6) bullet list, please check the Markdown style and the output style
  The markdown file looks like:
  image
  But the output is:
  image

**Comment by sebromero** [comment]
Any update on this?

**Comment by marqdevx** [comment]
@jcarolinares What is the status of this?

**Comment by marqdevx** [comment]
If possible, make clear the Latching compatible values

**Comment by jcarolinares** [comment]
Branch is so outdated that not even the old render script is working

I'm going to move the work done to a new branch cloned from main

**Comment by marqdevx** [comment]
@jcarolinares Check this branch
https://github.com/arduino/docs-content/compare/main...marqdevx/portenta-machine-control/datasheet-fixes

**Pull request closed without merging**